Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsHand.cs

```diff
@@ -15,59 +15,43 @@
 using System;
 using System.Linq;
 using System.Collections.Generic;
-using System.Collections;
+using Unity.Collections;
+using Unity.Jobs;
+using Unity.Jobs.LowLevel.Unsafe;
 
 namespace Leap.Unity.Interaction.PhysicsHands
 {
     public class PhysicsHand : MonoBehaviour
     {
-        [System.Serializable]
+        [Serializable]
         public class Hand
         {
             public const int FINGERS = 5, BONES = 3;
-            public const float TRIGGER_DISTANCE = 0.004f;
-            public const float CONTACT_ENTER_DISTANCE = 0.0015f, CONTACT_EXIT_DISTANCE = 0.01f;
-            public const float CONTACT_THUMB_ENTER_DISTANCE = 0.0035f, CONTACT_THUMB_EXIT_DISTANCE = 0.018f;
-            // Used as velocity * fixedDeltaTime
-            public const float MAXIMUM_PALM_VELOCITY = 300f, MINIMUM_PALM_VELOCITY = 50f, MAXIMUM_FINGER_VELOCITY = 200f, MINIMUM_FINGER_VELOCITY = 50f;
-            // Used as angularVelocity * fixedDeltaTime
-            public const float MAXIMUM_PALM_ANGULAR_VELOCITY = 8000f, MINIMUM_PALM_ANGULAR_VELOCITY = 6000f;
-
-            [Tooltip("The distance that bones will have their radius inflated by when calculating if an object is grabbed.")]
-            public float triggerDistance = TRIGGER_DISTANCE;
-            // You can change this to reduce the overall speed of the hands
-            [Tooltip("The velocity at which the hand will move when not contacting or grabbing any object. Reducing this number may result in additional hand latency.")]
-            public float maximumPalmVelocity = MAXIMUM_PALM_VELOCITY;
-            [Tooltip("The velocity that the hand will reduce down to, the further it gets away from the original data hand. " +
-                "Increasing this number will cause the hand to appear \"stronger\" when pushing into objects, if less stable.")]
-            public float minimumPalmVelocity = MINIMUM_PALM_VELOCITY;
-
-            public float maximumFingerVelocity = MAXIMUM_FINGER_VELOCITY;
-
-            public float minimumFingerVelocity = MINIMUM_FINGER_VELOCITY;
 
-            public float maximumPalmAngularVelocity = MAXIMUM_PALM_ANGULAR_VELOCITY;
-
-            public float minimumPalmAngularVelocity = MINIMUM_PALM_ANGULAR_VELOCITY;
+            /// <summary>
+            /// All of the adjustable parameters of the hand
+            /// </summary>
+            [HideInInspector]
+            public HandParameters parameters = new HandParameters();
 
             [HideInInspector]
```

Comment:
All the internals here are hidden in inspector by definition - can we remove the attribute?